### PR TITLE
chore: update pnpm lockfile

### PR DESCRIPTION
**Pull request by MustafaWael**

### CODE REVIEW

**Review by MustafaWael-work — APPROVED** [review]

**Review by MustafaWael — COMMENTED** [review]
thanks

**Review by MustafaWael-work — APPROVED** [review]
good

**Review by MustafaWael-work — APPROVED** [review]

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -68,7 +68,7 @@ importers:
         version: 8.4.35
       tailwindcss:
         specifier: ^3.4.1
-        version: 3.4.1
+        version: 3.4.1(ts-node@10.9.2)
```
Comment (MustafaWael-work) [inline pnpm-lock.yaml:71]: good

**Review by MustafaWael-work — APPROVED** [review]

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -117,7 +117,7 @@ importers:
...
```
Comment (MustafaWael-work) [inline pnpm-lock.yaml:120]: it's fine

Comment (MustafaWael-work) [reply pnpm-lock.yaml:120]: good

**Review by MustafaWael-work — APPROVED** [review]
good change

**Pull request merged**